web_src/css/modules/tippy.css

```diff
@@ -37,7 +37,7 @@
 
 .tippy-content {
   position: relative;
-  padding: 1rem;
+  padding: 0;
```

Comment:
This will likely have unintended side-effects on other tippies. We should likely set `theme: "menu"` on this tippy instead so that `[data-theme="menu"]` below gets into effect. As far as I'm aware this theme option can only be set from JS currently.

Reply:
Maybe this is still a concern? @yp05327

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/26284/commits/228eb7be860272363f0c283466bddf777dc54e25.